home-work-app/src/app/components/MoviesSection/MoviesSection.tsx

```diff
@@ -16,54 +16,54 @@ interface MoviesSectionProps extends MapDispatchToProps {
     moviesCollection: IApiResponse.IMovie;
 }
 
-const filterItems: FilterItems = AppConstants.filterItems;
+const { filterItems } = AppConstants;
 
-const mapStateToProps: MapStateToProps = (state: GlobalState): {
-    moviesCollection: IApiResponse.IMovie[]
+const mapStateToProps: MapStateToProps = (
+    state: GlobalState
+): {
+    moviesCollection: IApiResponse.IMovie[];
 } => ({
-    moviesCollection: state.movieEditor.movies
+    moviesCollection: state.movieEditor.movies,
 });
 
 const mapDispatchToProps: MapDispatchToProps = {
     deleteMovie: deleteMovieAction,
-    update: updateMovieAction
+    update: updateMovieAction,
 };
 
 const MoviesSectionComponent: React.FC<MoviesSectionProps> = ({
     moviesCollection,
     deleteMovie,
-    update
+    update,
 }: MoviesSectionProps) => {
     const [movies, setMovies] = useState(null);
```

Comment:
You don't need to set `movies` in internal state variable here. You can use `moviesCollection` directly.